handsontable/src/shortcuts/context.js

```diff
@@ -22,22 +22,21 @@ export const createContext = (name) => {
   });
 
   /**
-   * Add a keyboard shortcut to this `ShortcutContext`.
+   * Add a keyboard shortcut to this context.
```

Comment:
This JSDoc makes the linter fail.

Reply:
Thanks, fixed: [f4d4c66](https://github.com/handsontable/handsontable/pull/9271/commits/f4d4c66996f6572517de960557b989c9f8c84a9c)